.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,23 @@
+---
+name: "🚀 Feature request"
+about: "If you have a feature request 💡"
```

Comment:
Maybe something along the lines "Tell us what you are missing" is better - having feature request in name as well as in about seems redundant. :)

But I'm also okay with the current solution - just an idea :)

Reply:
@craftamap Good idea!